package.json

```diff
@@ -13,20 +13,26 @@
   "dependencies": {
     "@react-native-firebase/app": "^17.4.3",
     "@react-native-firebase/auth": "^17.4.3",
+    "@react-native-firebase/database": "^17.5.0",
     "@react-navigation/bottom-tabs": "^6.5.7",
     "@react-navigation/native": "^6.1.6",
     "@react-navigation/stack": "^6.3.16",
     "@reduxjs/toolkit": "^1.9.5",
     "axios": "^1.4.0",
+    "date-fns": "^2.30.0",
```

Comment:
you should try to look at dayjs - it's lightweight and offers as much functionality as date fns